data-loader/server.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask
+from app import controller
+import re
+import os
+import datetime
+
+facebook_data_root = "../data/derived/fib_results/facebook/"
+twitter_data_root = "../data/derived/fib_results/twitter/"
+PLATFORMS = ["Facebook", "Twitter"]
+LOAD_PAST_MONTH_DATA = ['2023_01']
+
+def load_current_data():
+    """
+    Load each platform and read the files.
+    """
+    for platform in PLATFORMS:
+        if platform == 'Facebook':
+            read_dir = facebook_data_root
+        else:
+            read_dir = twitter_data_root
+        all_year_month = get_available_year_month(read_dir)
+        current_year_month = datetime.datetime.now().strftime("%Y_%m")
+        selected_month = list(filter(lambda x: current_year_month in x, all_year_month))
+        if len(selected_month) == 1:
+            controller.read_files(read_dir, platform, selected_month[0])
+        else:
+            raise Exception("Error in finding data on", current_year_month)
+            logger.error("Error in finding data on {}", current_year_month)
+
+def load_past_data():
+    """
+    Load each platform and read the files.
+    """
+    for selected_month in LOAD_PAST_MONTH_DATA:
```

Comment:
It looks like `LOAD_PAST_MONTH_DATA` is a list of one string. Is this what we want? it seems like we would need to update this manually each month. This is not what we want as it seems like it will defeat the purpose of the cronjob. Shouldn't we instead want to just automatically create the `YYYY_mm` string for the past month with something like `datetime`?

Reply:
This LOAD_PAST_MONTH data is used to upload the previous months data as an array. (2022_01,2022_02 ... to 2023_01), So I kept this, because there is another month remained to upload for the 2022_02 data.